src/expedited.cpp

```diff
@@ -0,0 +1,289 @@
+// Copyright (c) 2015-2017 The Bitcoin Unlimited developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "expedited.h"
+#include "main.h"
+#include "rpc/server.h"
+#include "thinblock.h"
+#include "tweak.h"
+#include "unlimited.h"
+
+#include <univalue.h>
+
+#define NUM_XPEDITED_STORE 10
+
+// Just save the last few expedited sent blocks so we don't resend (uint256)
+uint256 xpeditedBlkSent[NUM_XPEDITED_STORE];
+
+// zeros on construction)
+int xpeditedBlkSendPos = 0;
+
+using namespace std;
+
```

Comment:
Is the using needed?  I see std:: throughout the file; let's be consistently explicit as that seems to be the trend.